esmf_regrid/schemes.py

```diff
@@ -299,6 +301,8 @@ def _map_complete_blocks(src, func, active_dims, out_sizes, *args, **kwargs):
         Dimensions that cannot be chunked.
     out_sizes : tuple of int
         Output size of dimensions that cannot be chunked.
+    dtype : type, optional
+        Type of the output array, if not given, the dtype of src is used.
```

Comment:
Actually no. after offline discussion I think this is a bad suggestion.
Since effectively \*args and \**kwargs contain additional arguments to the wrapped array function, and so should probably *not* contain extra controls for `map_blocks`, since that is a confusingly mixed purpose !

So, I now think this is **best left as it is**.